config/cluster.yaml

```diff
@@ -0,0 +1,14 @@
+##  cluster settings
+# id: <id of the cluster to be created>
+# vm_size: <vm-size, see available options here: https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/>
+# size: <number of dedicated nodes in the cluster, not that clusters must contain all dedicated or all low priority nodes>
+# size_low_pri: <number of low priority nodes in the cluster, mutually exclusive with size setting>
+# username: <username for the linux user to be created> (optional)
+# custom_script: <path to custom script to run on each node> (optional)
```

Comment:
we should add docker-repo as a variable too. we can do "azb spark cluster create ... --docker-reop <repo>"

Reply:
Is that in the scope of this PR?

Reply:
good point, might not be. or might want to do it when you merge. not sure what the best practice for this is.